practice-one/src/pages/Analytics/Analytics.jsx

```diff
@@ -1,7 +1,68 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import "./Analytics.css";
+import { Button, Expand, CustomerItem } from "@components";
+import { BUTTON_VARIANTS } from "@constants/buttons";
+import { plusIcon } from "@assets/images";
+import { EXPAND_TITLES } from "@data";
+import { getAllCustomerService } from "../../services/customerService";
+import { v4 as uuidv4 } from "uuid";
 
 const Analytics = () => {
-  return <h1>Analytic</h1>;
+  const [customers, setCustomers] = useState([]);
+
+  useEffect(() => {
+    const getAllCustomers = async () => {
+      // Call getAllCustomerService function to get all data
+      const data = await getAllCustomerService();
```

Comment:
how about the error case?

Reply:
I updated error case also empty case
![image](https://github.com/datdragon777/React-Training/assets/74806826/50fa1af1-0047-439f-8abe-311a3bed7e1c)
![image](https://github.com/datdragon777/React-Training/assets/74806826/02cf6455-66ec-41fb-b238-1a4d7faf85df)